src/video/khronos/EGL/eglplatform.h

```diff
@@ -48,13 +48,13 @@
  * implementations.
  */
 
-#ifdef EGL_NO_PLATFORM_SPECIFIC_TYPES
+#if defined(EGL_NO_PLATFORM_SPECIFIC_TYPES)
```

Comment:
What I'm trying to do is return this file to the way Khronos does it. Specifically, I'm returning it to the content it had in commit 23ffa3d9 "Update SDL's copy of the EGL headers from Khronos".

Since then, it was swapped from `#if defined(...)` to `#ifdef` in commit b6ae281e "Use #ifdef/#ifndef instead of #if defined/#if \!defined". If you're saying that b6ae281e shouldn't have touched this file, then, yes, I agree, and that's why I'm reverting that change too.

Reply:
I see, OK then!

Reply:
I've checked that after this PR, `src/video/khronos/EGL/` will be very similar to the `/usr/include/EGL/` on my Debian system. It's slightly different because SDL's copy is a little older (20211210 rather than 20220525) but the coding style, etc. is the same.